2024-10-09 01:16:45,450 INFO    MainThread:80800 [wandb_setup.py:_flush():79] Current SDK version is 0.18.3
2024-10-09 01:16:45,450 INFO    MainThread:80800 [wandb_setup.py:_flush():79] Configure stats pid to 80800
2024-10-09 01:16:45,450 INFO    MainThread:80800 [wandb_setup.py:_flush():79] Loading settings from /home/autrio/.config/wandb/settings
2024-10-09 01:16:45,450 INFO    MainThread:80800 [wandb_setup.py:_flush():79] Loading settings from /home/autrio/college-linx/SMAI/smai-m24-assignments-Autrio/wandb/settings
2024-10-09 01:16:45,450 INFO    MainThread:80800 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-09 01:16:45,450 INFO    MainThread:80800 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-09 01:16:45,450 WARNING MainThread:80800 [wandb_setup.py:_flush():79] Could not find program at -m assignments.3.2.2.2
2024-10-09 01:16:45,450 INFO    MainThread:80800 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program_relpath': None, 'program': '-m assignments.3.2.2.2'}
2024-10-09 01:16:45,450 INFO    MainThread:80800 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-09 01:16:45,450 INFO    MainThread:80800 [wandb_init.py:_log_setup():532] Logging user logs to /home/autrio/college-linx/SMAI/smai-m24-assignments-Autrio/wandb/run-20241009_011645-jvx4beo2/logs/debug.log
2024-10-09 01:16:45,450 INFO    MainThread:80800 [wandb_init.py:_log_setup():533] Logging internal logs to /home/autrio/college-linx/SMAI/smai-m24-assignments-Autrio/wandb/run-20241009_011645-jvx4beo2/logs/debug-internal.log
2024-10-09 01:16:45,450 INFO    MainThread:80800 [wandb_init.py:init():617] calling init triggers
2024-10-09 01:16:45,450 INFO    MainThread:80800 [wandb_init.py:init():624] wandb.init called with sweep_config: {}
config: {'lr': 0.001, 'batch_size': 100, 'epoch': 100, 'optimizer': None, 'loss_fn': <models.MLP.MLP2.Loss.MSELoss object at 0x7f196f31d690>, 'activation': <models.MLP.MLP2.activations.Tanh object at 0x7f196f2d4040>, 'type': 'classification'}
2024-10-09 01:16:45,450 INFO    MainThread:80800 [wandb_init.py:init():667] starting backend
2024-10-09 01:16:45,450 INFO    MainThread:80800 [wandb_init.py:init():671] sending inform_init request
2024-10-09 01:16:45,451 INFO    MainThread:80800 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-09 01:16:45,451 INFO    MainThread:80800 [wandb_init.py:init():684] backend started and connected
2024-10-09 01:16:45,453 INFO    MainThread:80800 [wandb_init.py:init():779] updated telemetry
2024-10-09 01:16:45,455 INFO    MainThread:80800 [wandb_init.py:init():812] communicating run to backend with 90.0 second timeout
2024-10-09 01:16:45,869 INFO    MainThread:80800 [wandb_init.py:init():863] starting run threads in backend
2024-10-09 01:16:46,057 INFO    MainThread:80800 [wandb_run.py:_console_start():2465] atexit reg
2024-10-09 01:16:46,057 INFO    MainThread:80800 [wandb_run.py:_redirect():2313] redirect: wrap_raw
2024-10-09 01:16:46,057 INFO    MainThread:80800 [wandb_run.py:_redirect():2378] Wrapping output streams.
2024-10-09 01:16:46,057 INFO    MainThread:80800 [wandb_run.py:_redirect():2403] Redirects installed.
2024-10-09 01:16:46,058 INFO    MainThread:80800 [wandb_init.py:init():907] run started, returning control to user process
2024-10-09 01:17:17,457 WARNING MsgRouterThr:80800 [router.py:message_loop():77] message_loop has been closed
